src/locale/translations/fr.json

```diff
@@ -52,8 +52,12 @@
         "ChooseRegionCTA": "Sélectionner une province ou un territoire"
       },
       "RegionCovered": {
-        "Title": "Vous n’avez pas de numéro à 8\u00A0chiffres?  ",
-        "Body": "Contactez la personne ou le bureau qui vous a donné vos résultats afin d’obtenir un numéro."
+        "ON": {
+          "Title": "Obtenir un numéro à 8 chiffres",
+          "Body": "Si vous avez reçu un test positif en Ontario :\n\n1. Connectez-vous au site Web des Résultats du test de diagnostic de la COVID-19\n\n2.Allez à la page de résultats de votre test.\n\n3.Copiez le numéro à 8 chiffres.\n\n4.Revenez ici et entrez le numéro.",
+          "CTA": "Aller sur le site des Résultats du test pour la COVID-19",
+          "Link": "https://covid19results.ehealthontario.ca:4443/login"
```

Comment:
Should this be the same as the en link?

Reply:
Not sure, just copied them over from figma

Reply:
Looks like they redirect to the same page anyways